packages/viewer/src/lib/scenes/camera.ts

```diff
@@ -162,14 +138,20 @@ export abstract class Camera {
    *
    * @param boundingBox - The bounding box to determine distance from.
    */
-  public distanceToBoundingBoxCenter(
+  public signedDistanceToBoundingBoxCenter(
     boundingBox?: BoundingBox.BoundingBox
   ): number {
-    const box = boundingBox || this.boundingBox;
-    const boundingBoxCenter = BoundingBox.center(box);
-    const cameraToCenter = cameraToVector(boundingBoxCenter, this.data);
+    const { position, viewVector } = withPositionAndViewVector(this.data);
+
+    const boundingBoxCenter = BoundingBox.center(
+      boundingBox ?? this.boundingBox
+    );
+    const cameraToCenter = Vector3.subtract(position, boundingBoxCenter);
 
-    return distanceToVectorAlongViewVec(cameraToCenter, this.data);
+    return (
+      Math.abs(Vector3.dot(viewVector, cameraToCenter)) /
```

Comment:
The `abs()` is still here!  :)

Reply:
Shoot, I missed removing it here and added it to the interaction hander... I'll fix that